draft-gpew-priv-ppm.md

```diff
@@ -379,6 +379,7 @@ in the "type" field (within the PPM URN namespace "urn:ietf:params:ppm:error:"):
 | unrecognizedTask        | An endpoint received a message with an unknown task ID. |
 | outdatedConfig          | The message was generated using an outdated configuration. |
 | batchInvalid            | A collect or aggregate-share request was made with invalid batch parameters. |
+| invalidReportShare      | The report share was invalid and could not be processed. |
```

Comment:
This error type doesn't seem to be used anywhere.

Reply:
Leftover 👍